README.md

```diff
@@ -1,10 +1,63 @@
 ## angular-filemanager
 File manager developed with AngularJS and Bootstrap by [Jonas Sciangula Street](https://github.com/joni2back)
 
+### use in your project
+
+#### 1) install and use
+1) bower install --save angular-filemanager
+2) include the dependencies in your project:
+
+    <!-- third party -->
+    <script src="/bower_components/angular-translate/angular-translate.min.js"></script>
+    <script src="/bower_components/angular-cookies/angular-cookies.min.js"></script>
+    
+    <!-- angular-filemanager -->
+    <link rel="stylesheet" href="/bower_components/angular-filemanager/dist/angular-filemanager.css">
+    <script src="/bower_components/angular-filemanager/dist/angular-filemanager.min.js"></script>
```

Comment:
ah yes, sorry. We must register your angular-filemanager first in the bower registry via commandline so bower can find the repo. To do that, just checkout my branch and run `bower register angular-filemanager git://github.com/joni2back/angular-filemanager.git`. After that the angular project is available at the registry `http://bower.io/search` and other users can install it via `bower install --save angular-filemanager`.